userspace/libsinsp/threadinfo.h

```diff
@@ -314,6 +314,21 @@ class SINSP_PUBLIC sinsp_threadinfo
 	//
 	sinsp *m_inspector;
 
+public: // types required for use in sets
+	struct hasher {
```

Comment:
hasher -> hash, maybe

Reply:
it's not the hash, though. it's only the thing that computes it.